pkg/forwarder/transaction/transaction.go

```diff
@@ -384,3 +384,18 @@ func (t *HTTPTransaction) SerializeTo(serializer TransactionsSerializer) error {
 	log.Trace("The transaction is not stored on disk because `storableOnDisk` is false.")
 	return nil
 }
+
+// readResponseBody read HTTP response body. If body is more than 1000 bytes
+// truncates it to  prevent from logging a huge message.
+// 1000 bytes are enough to tell which tools a response comes.
+func readResponseBody(body io.ReadCloser) ([]byte, error) {
+	b, err := io.ReadAll(body)
+	if err != nil {
+		return nil, err
+	}
+	firstBytes := 1000
+	if firstBytes < len(b) {
+		b = b[:firstBytes]
+	}
```

Comment:
I think some sort of a warning or debug message (depending on how you view it) may be appropriate if we are truncating any part of the server response.